pl_examples/pytorch_ecosystem/pytorch_geometric/pyproject.toml

```
@@ -0,0 +1,25 @@
+[tool.poetry]
+name = "lightning-geometric"
+version = "0.1.0"
+description = "TorchScripted Pytorch Geometric Examples with Pytorch Lightning"
+authors = ["Thomas Chaton <thomas.ai@grid.com>"]
```

still the mail here...
```suggestion
authors = ["Thomas Chaton <thomas@pytorchlightning.ai>"]
```